src/System.Private.ServiceModel/tools/PRService/pr.ashx

```diff
@@ -273,6 +349,62 @@ public class PullRequestHandler : IHttpHandler
         return success;
     }
 
+    // Read configuration
+    // Precedence is: 
+    // 1. Environment variable
+    // 2. AppSettings in web.config
+    // 
+    private bool ReadConfiguration(StringBuilder result)
```

Comment:
Nit: not blocking, but could add comment saying what 'bool' return and StringBuilder parameter are used for.  I interpret 'result' to be the error result whenver it fails.